akka-docs/src/test/java/jdocs/circuitbreaker/DangerousJavaActor.java

```diff
@@ -6,14 +6,13 @@
 
 // #imports1
 
+import static akka.pattern.Patterns.pipe;
+
 import akka.actor.AbstractActor;
+import akka.event.Logging;
 import akka.event.LoggingAdapter;
-import java.time.Duration;
 import akka.pattern.CircuitBreaker;
-import akka.event.Logging;
-
-import static akka.pattern.Patterns.pipe;
-
+import java.time.Duration;
 import java.util.concurrent.CompletableFuture;
 
 // #imports1
```

Comment:
At first I thought there was a high risk that this reformat would break the doc snippets for imports, but after trying to review it I can't any place where we have that kind of "selective" import snippets for Java.